Documentation/qa-duty.md

```diff
@@ -37,6 +37,22 @@ The general advise is to do a short investigation before asking for help from
 others in the team. For obvious test infra errors we can just re-run the jobs;
 for actual test failures we need to asses if it might be a regression or not.
 
+### Nightly Mender Server Integration tests
+
+Link: *qastatus link pending*
+Link: https://gitlab.com/Northern.tech/Mender/integration/-/pipeline_schedules
+Link: https://gitlab.com/Northern.tech/Mender/integration/-/pipelines?page=1&scope=all&source=schedule
+ 
+These tests are related to the Mender Server monorepo, tesing it over the current
+stable Mender client artifacts.  
```

Comment:
At the moment, we are testing Mender Server `main` with with Mender client artifacts `master`.